treeherder/extract/extract_perf.py

```diff
@@ -0,0 +1,120 @@
+from redis import Redis
+
+from jx_bigquery import bigquery
+from jx_mysql.mysql import (MySQL,
+                            sql_query)
+from jx_mysql.mysql_snowflake_extractor import MySqlSnowflakeExtractor
+from mo_files import File
+from mo_json import (json2value,
+                     value2json)
+from mo_logs import (Log,
+                     constants,
+                     startup)
+from mo_sql import SQL
+from mo_times import Timer
+from treeherder.extract import VENDOR_PATH
+
+CONFIG_FILE = (File.new_instance(__file__).parent / "extract_perf.json").abspath
+
+_keep_import = VENDOR_PATH
+
+
+class ExtractPerf:
+    def run(self, force=False, restart=False, merge=False):
+        # SETUP LOGGING
+        settings = startup.read_settings(filename=CONFIG_FILE)
+        constants.set(settings.constants)
+        Log.start(settings.debug)
+
+        if not settings.extractor.app_name:
+            Log.error("Expecting an extractor.app_name in config file")
```

Comment:
s/Log/Django Logger :)

Reply:
@camd  Maybe I am missing something.  Shoudl I be using `logging.getLogger(__name__)` or something else?

Reply:
using `__name__` is how I usually do it.  But any name that makes  sense is totally fine.  I suppose the advantage to `__name__` is that it makes it explicit which file to look in when an error happens.

Reply:
@camd Thank you.  The `Log` library is responsible collecting stack traces and metadata for the error, so the `__name__` is superfluous